community/installation-guides/panel/ubuntu2004.md

```diff
@@ -0,0 +1,115 @@
+# Ubuntu 20.04
+In this guide we will install Pterodactyl 0.7.X — including all of it's dependencies — and configure our webserver to serve it using SSL.
+
+[[toc]]
+
+::: tip
+This guide is based off the [official installation documentation](/panel/getting_started.md) but is tailored specifically for Ubuntu 20.04.
+:::
+
+## Install Requirements
+We will first begin by installing all of Pterodactyl's [required](/panel/getting_started.md#dependencies) dependencies.
+
+### MariaDB
+```bash
+## Get apt updates
+apt update -y
+
+## Install MariaDB 10.1
+apt install -y mariadb-common mariadb-server mariadb-client
+
+## Start MariaDB
+systemctl start mariadb
+systemctl enable mariadb
+```
+
+### PHP 7.4
+
+```bash
+## Get apt updates
+apt update -y
+
+## Install PHP 7.4
+apt install -y php7.4 php7.4-cli php7.4-gd php7.4-mysql php7.4-pdo php7.4-mbstring php7.4-tokenizer php7.4-bcmath php7.4-xml php7.4-fpm php7.4-curl php7.4-zip
+```
+
+### Nginx
+
+```bash
+apt install -y nginx
+```
+
+### Redis
+
+```bash
+apt install -y redis-server
+
+systemctl start redis-server
+systemctl enable redis-server
```

Comment:
Could remove these as they're at the end of making the pteroq.service file in the main docs, that you direct them to go once done with the redis setup at the bottom

Reply:
Commands are different between between Cent and Ubuntu I call it out now out of habit. Still good to have around.

Reply:
You could also change it to `systemctl enable --now redis-server` to run both `start` and `enable` in one command.

Reply:
As this is specifically for Ubuntu 20.04... I see nothing wrong with using `systemctl enable --now <service>` for the services

Reply:
As previously stated.

![image](https://user-images.githubusercontent.com/1207679/80266980-d23ca080-866c-11ea-91f1-18cbd9df504d.png)